our_land/templates/profile/profile_detail.html

```diff
@@ -0,0 +1,147 @@
+{% extends 'profile/profile_detail.html' %} {% load humanize lookup staticfiles %}
+
+{% block secondary_nav %}
+    <div class="floating-nav-wrapper" id="sticky-nav">
+      <nav class="floating-nav affix-top" data-spy="affix" data-offset-top="197">
+        <div class="wrapper clearfix">
+          <div class="geography-ico">
+                    <span><img
+                        src="{% static 'img/icons/favicon.ico' %}">{{ geography.this.name }}
+                    </span>
+          </div>
+          <ul>
+            <li><a href="#firm-land"><i class="fa fa-check"></i>
+              <span>Firms & Agricultural Land</span></a></li>
+            <li><a href="#erven-land"><i class="fa fa-home"></i>
+              <span>Erven & Urban Land</span></a></li>
+            <li><a href="#sectional-title-land"><i class="fa fa-user"></i>
+              <span>Sectional Title Land</span></a></li>
+          </ul>
+        </div>
+      </nav>
+    </div>
+{% endblock %}
+{% block profile_header_population %}
+{% endblock %}
+
+{% block profile_header_tail %}
+{% endblock %}
+
+{% block profile_detail %}
+  <article id="firm-land" class="clearfix">
```

Comment:
See [Takwimu](https://github.com/TakwimuAfrica/TAKWIMU/blob/dc66680d0b15048acee146c8aa67f2ed9d32ea5d/takwimu/templates/profile/profile_detail.html#L38) on how to put guards around templates so that the chart is not visible if data is missing.